x-pack/solutions/security/plugins/security_solution/public/common/utils/integrations/index.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { APP_UI_ID } from '../../../../common';
+
+export const RETURN_APP_ID = 'returnAppId';
+export const RETURN_PATH = 'returnPath';
+
+export const addPathParamToUrl = (url: string, onboardingLink: string) => {
```

Comment:
Can we use [query-string](https://www.npmjs.com/package/query-string)or some other lib for this? 

also a good candidate for a couple unit tests because why not :)

Reply:
Just seen this is copied from elsewhere, but still! :)